eclair-core/src/main/scala/fr/acinq/eclair/wire/ChannelCodecs.scala

```diff
@@ -269,6 +269,7 @@ object ChannelCodecs extends Logging {
       ("localCommitPublished" | optional(bool8, localCommitPublishedCodec)) ::
       ("remoteCommitPublished" | optional(bool8, remoteCommitPublishedCodec)) ::
       ("nextRemoteCommitPublished" | optional(bool8, remoteCommitPublishedCodec)) ::
+      ("customRemoteCommitPublished" | provide(Map.empty[RemoteCommit, RemoteCommitPublished])) ::
```

Comment:
The trick is that when we restart, we forget everything about the custom remote commit, but that's not an issue because:

1. We don't have anything to claim on our side (thanks to `static_remotekey`)
2. We will re-create it automatically because we will watch the funding output being spent

Is that correct? That probably deserves a comment, it's quite subtle.

Reply:
That's correct, will add a comment.

Reply:
Done with 794c75fb8.